packages/synapse-react-client/src/components/FullTextSearch/FullTextSearchUtils.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  ColumnModel,
+  ColumnSingleValueFilterOperator,
+  ColumnSingleValueQueryFilter,
+  QueryBundleRequest,
+  TextMatchesQueryFilter,
+} from '@sage-bionetworks/synapse-types'
+import React from 'react'
+import { getFileColumnModelId } from '../SynapseTable/SynapseTableUtils'
+import { SYNAPSE_ENTITY_ID_REGEX } from '../../utils/functions/RegularExpressions'
+
+export function updateQueryUsingSearchTerm(
```

Comment:
Extracted from `FullTextSearch` component body, and tested in `FullTextSearch.test.tsx`

Reply:
Thank you for including the changes from [this recent PR](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/1142/files#diff-a587ebd1faa5d3610e81c778f3b1329519c0b631585e3874bd1375256179e1a2)!